dll/win32/cabinet/fdi.c

```diff
@@ -2050,24 +2050,13 @@ static int fdi_decomp(const struct fdi_file *fi, int savemode, fdi_decomp_state
             fullpath[0] = '\0';
             if (pathlen) {
               strcpy(fullpath, userpath);
-#ifdef __REACTOS__
-              if (fullpath[pathlen - 1] == '\\')
-                fullpath[pathlen - 1] = '\0';
-#else
```

Comment:
Those blocks were once introduced for [ROSTESTS-135](https://jira.reactos.org/browse/ROSTESTS-135)

So if your PR would put a green checkmark after your undone TODO:
![image](https://github.com/user-attachments/assets/67977fa0-2407-4527-bb15-c4ce98ba777d)
that would definitely help strengthening our trust into that part of the PR